phil/what-is-master.md

```diff
@@ -1,8 +1,37 @@
-# What is master?
+# What is the `/master` Branch?
 
 Currently, it is "skimmed-over code" (not necessarily tested).
 
-**Proposal:** It is tested code (at least lightly).
+**Proposal:** It is tested code (at least lightly by code reviewer or QA).
+
+**Note:** We _should_ begin using manifestly properly; this document is only about the label of the bottom row in [JP's diagram](https://www.dropbox.com/s/uwp0jq8luo51uqw/release_candidates_v2.pdf?dl=0). I think a deploy to any env should add an entry to [deploy-manifests](https://github.com/openstax/deploy-manifests).
+
+**Note:** If this is scrapped, I think:
+
+- devs/leads still need to deploy `git fetch origin pull/${PR_NUMBER}/merge`
+- it should be revisited when selenium tests cover most of the testrails tests
+
+# Problem
+
+Frequently, code that is deployed to dev/qa/production is **not** on `/master`. A "hotfix" is the common example for production.
+
+1. Code that is in `/master` should have already been tested <small>(I think by someone other than the primary developer)</small>
+  - If the code is in `/master` it's too late to discuss if the feature should go out
+  - People that "Merge" the code frequently read but do not deploy it anywhere to test. The _only_ testing is done by the original developer.
```

Comment:
Sometimes that original developer's testing is automated and so reading of the spec is something of an outside test at least for us (I'm bad at this too, though).  Also, for BE it is sometimes difficult to test in a non-spec way.